utils/auth.util.js

```diff
@@ -1,20 +1,23 @@
-const bcrypt = require("bcrypt");
-const jwt = require("jsonwebtoken")
-const generateJwtToken = async (Username)=>{
-  const token = jwt.sign({
-      username: Username,
-    }, process.env.JWT_SECRET_TOKEN, {
-      algorithm: process.env.JWT_SECRET_TOKEN_ALGORITHM,
-      expiresIn: process.env.JWT_SECRET_TOKEN_EXPIRE_TIME
-    });
-  return token;
-}
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
 
-
-const comparePassword = async (inputPassword,userPassword) => {
-  return await bcrypt.compare(inputPassword,userPassword);
-  };
+const generateJwtToken = async (Username) => {
+    const token = jwt.sign(
+        {
+            username: Username,
+        },
+        process.env.JWT_SECRET_TOKEN,
+        {
+            algorithm: process.env.JWT_SECRET_TOKEN_ALGORITHM,
+            expiresIn: process.env.JWT_SECRET_TOKEN_EXPIRE_TIME,
+        }
+    );
+    return token;
+};
+// eslint-disable-next-line no-return-await, prettier/prettier
+const comparePassword = async (inputPassword, userPassword) => await bcrypt.compare(inputPassword, userPassword);
```

Comment:
if you are going to return an asynchronous function. It doesn't require `await` keyword as it will return promise.